inspirehep/modules/workflows/tasks/magpie.py

```diff
@@ -120,17 +120,23 @@ def guess_experiments(obj, eng):
     """Workflow task to ask Magpie API for a subject area assessment."""
     magpie_url = get_magpie_url()
     if not magpie_url:
-        # Skip task if no API URL set
+        # Skip task if no magpie API URL set
         return
 
     payload = prepare_magpie_payload(obj.data, corpus="experiments")
     results = json_api_request(magpie_url, payload)
     if results:
-        labels = results.get('labels', [])
-        experiments = filter_magpie_response(labels, limit=0.5)
+        all_predictions = results.get('labels', [])
+        selected_experiments = filter_magpie_response(
+            all_predictions,
+            limit=0.5,
+        )
+        selected_experiments = [
+            {'label': e[0], 'score': e[1]}
+            for e in selected_experiments
+        ]
         obj.extra_data["experiments_prediction"] = dict(
-            labels=labels,
-            experiments=experiments
+            experiments=selected_experiments,
```

Comment:
I've copied the `categories_prediction` for the format. I think this is not used anywhere though.